README.md

```diff
@@ -27,4 +27,21 @@ netgen_ibexa_admin_ui_extra:
     resource: '@NetgenIbexaAdminUIExtraBundle/Resources/config/routing.yaml'
 ```
 
+Content URLs by Siteaccess
+--------------------------
+
+This package enhances the visibility of Content URLs by Siteaccess. URLs can be viewed in the administration interface under the **URL** tab within the Content view.
+
+The package distinguishes between two types of URLs:
+
+1. **Content Tree URLs**: URLs that reside within the configured Siteaccess Content tree.
+2. **External URLs**: URLs that exist outside of the configured Siteaccess Content tree.
+
+By default, the overview of External URLs is hidden. To enable the display of these URLs, set the following parameter in your configuration:
+
+```yaml
+netgen_ibexa_admin_ui_extra.show_external_siteaccess_urls: true
```

Comment:
This parameter is internal, what's exposed to the users is bundle configuration, so document that instead.

Reply:
fixed in https://github.com/netgen/ibexa-admin-ui-extra/pull/2/commits/a3b8b53a2be9b78bab0c21415f54dd6809393da0